tests/kola/files/trust-cpu

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# kola: { "exclusive": false }
+
+# This is RHCOS only
+# https://github.com/coreos/fedora-coreos-tracker/issues/1369
+# https://github.com/coreos/fedora-coreos-config/pull/2155
```

Comment:
maybe mention that this test will start failing and can be removed when RHCOS gets to a 6.2+ kernel.

Reply:
updated!